### DISCUSSION TITLE
[Feature] pCloud support?

### DISCUSSION THREAD

**Discussion by Neustradamus** [immich-app/immich#5743] [Feature Request]
### The feature

Dear @immich-app team,

It is possible to add pCloud support?

Thanks in advance.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by hmalaud** [comment]
Would be incredibly neat

**Comment by ahmadyusuf** [comment]
I hope this can be  added as well

**Comment by MaleicAcid** [comment]
+1 really need this

**Comment by bo0tzz** [comment]
We have no plans to support arbitrary file storage platforms. If you can mount it locally that might work, though expect poor performance.

**Comment by Neustradamus** [comment]
@bo0tzz: It is not good to close unsolved ticket, can you reopen?

**Reply by bo0tzz** [reply]
I closed it because we won't support pcloud, as I said in my message above.
